### ISSUE TITLE
 How do I remove the Softmax layer from the model structure

### ISSUE THREAD

**Issue by yanni1995** [open-mmlab/mmsegmentation#1341]
When I changed the structure in Encoder-Decoder, SoftMax was still there.
 assert self.test_cfg.mode in ['slide', 'whole']
        ori_shape = img_meta[0]['ori_shape']
        assert all(_['ori_shape'] == ori_shape for _ in img_meta)
        if self.test_cfg.mode == 'slide':
            seg_logit = self.slide_inference(img, img_meta, rescale)
        else:
            seg_logit = self.whole_inference(img, img_meta, rescale)
        # output = F.softmax(seg_logit, dim=1)
        output = seg_logit

Please reply as soon as possible.

**Comment by MeowZheng** [comment]
Please take a look, this https://github.com/open-mmlab/mmsegmentation/pull/1374 might help
